src/sentry/integrations/vsts/integration.py

```diff
@@ -361,6 +360,20 @@ def get_oauth_data(self, payload):
 
         return data
 
+    def get_base_url(self, access_token, account_id):
+        session = http.build_session()
+        url = 'https://app.vssps.visualstudio.com/_apis/resourceareas/79134C72-4A58-4B42-976C-04E7115F32BF?hostId=%s&api-preview=5.0-preview.1' % account_id
```

Comment:
Maybe we can move this URL into a constant with some sort of descriptive name? Like `VSTS_ACCOUNT_LOOKUP_URL` or something. It's pretty opaque and hidden down here.